src/lib.rs

```diff
@@ -116,7 +116,7 @@
 use std::future::Future;
 
 use configuration::Configuration;
-pub use dogstatsd::{EventOptions, ServiceCheckOptions, ServiceStatus};
+pub use dogstatsd::{EventAlertType, EventOptions, ServiceCheckOptions, ServiceStatus};
```

Comment:
Can you please export both `EventPriority` and `EventAlertType` enums?

Reply:
Uh you are right, missed it